src/components/postdetail/postdetail.jsx

```diff
@@ -0,0 +1,40 @@
+import './postdetail.css'
+import {Link, useParams} from "react-router-dom";
+import Calctime from "../helpers/calctime.jsx";
+import {useEffect, useState} from "react";
+import axios from 'axios'
+
+function Postdetail() {
+    const {id} = useParams();
+
+    const [post, setPost] = useState([])
+
+    useEffect(() => {
+        fetchPost()
+    });
+
+    async function fetchPost() {
+        try {
+            const response = await axios.get(`http://localhost:3000/posts/${id}`)
+            setPost(response.data)
+        } catch (e) {
+            alert('Het ophalen van één specifieke post is mislukt.')
+        }
+    }
+
+    return (
```

Comment:
Zou deze code ook alleen terug geven als je post data niet leeg is. 
return (data && <div className='blog-container'> ... </div>)